en_US.json

```diff
@@ -782,7 +782,7 @@
 	"edgeHostname.tooltip.client.zone.servergroup": "Select the highest server group to use.  Your traffic will be served by all PoPs not higher than it.",
 	"edgeHostname.tooltip.client.zone.weight": "Adjust how this rule behaves relative to other rules for the same client zone.",
 	"edgeHostname.tooltip.server.regions": "Select the region(s) that can serve the edge hostname.",
-	"edgeHostname.update.accepted.modal.content": "The edge hostname update has been accepted. It will take a few seconds before the change takes effect. You can check the status in the {history} dropdown list.",
+	"edgeHostname.update.accepted.modal.content": "The edge hostname update has been accepted. It will take a few minutes before the change takes effect. You can check the status in the {history} dropdown list. Please check that the status ‘Succeeded’ and that the change has taken effect, before using the edge hostname in your DNS records.",
```

Comment:
Sorry for the confusion. Add the word "is":

 **Please check that the status is ‘Succeeded’**